src/library/scala/collection/immutable/Vector.scala

```diff
@@ -88,7 +83,7 @@ override def companion: GenericCompanion[Vector] = Vector
 
   override def lengthCompare(len: Int): Int = length - len
 
-  private[collection] final def initIterator[B >: A](s: VectorIterator[B]) {
+  private[collection] def initIterator[B >: A](s: VectorIterator[B]) {
```

Comment:
Ah.  No.  Sorry, I missed that.  I had somehow assumed, without any evidence, that it was in VectorPointer.